instrumentation/othttp/transport_example_test.go

```diff
@@ -15,45 +15,13 @@
 package othttp
 
 import (
-	"fmt"
-	"io/ioutil"
-	"log"
 	"net/http"
-
-	"go.opentelemetry.io/otel/api/global"
-	sdktrace "go.opentelemetry.io/otel/sdk/trace"
 )
 
 func ExampleNewTransport() {
-	// Start with a working trace provider
-	tp, err := sdktrace.NewProvider(sdktrace.WithConfig(sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}))
-	if err != nil {
-		log.Fatal(err)
-	}
-	global.SetTraceProvider(tp)
-
 	// Create an http.Client that uses the othttp.Transport
 	// wrapped around the http.DefaultTransport
-	client := http.Client{
+	_ = http.Client{
```

Comment:
This isn't a very useful example anymore, perhaps it would be best to simply delete it or move it into a `README.md`.

Reply:
This is going to be included in the godocs and I think it is a decent example of how to use it for a new users:

![20200728_143741](https://user-images.githubusercontent.com/5543599/88724882-12824380-d0e0-11ea-95dd-cea6f36a4abd.png)

Additionally, with it actually existing as Go code it is validated with the compiler resulting in valid code. This is something I noticed was not true in some other places where we tried to keep code in docs ([1](https://github.com/open-telemetry/opentelemetry-go/blob/master/instrumentation/grpctrace/interceptor.go#L68-L71), [2](https://github.com/open-telemetry/opentelemetry-go/blob/master/instrumentation/grpctrace/interceptor.go#L248), ...)

Reply:
That said, I'm not super tied to this example if we want to remove or expand it.

Reply:
I suppose that does remain a decent example.  Out of context it looks off, but I'm not against keeping it.